hack/generate-kustomize-patches.sh

```diff
@@ -38,7 +38,7 @@ docker run --rm manifest cat /file >"$KUSTOMIZE_INPUT_DIR/$release_asset_filenam
 
 # Update kustomize patches for webhooks. We do this for every CRD
 
-watch_filter="$(yq e '.watchfilter' "$helm_values")"
+watch_filter="$(yq e -e '.watchfilter' "$helm_values")" || { echo "Could not find watch filter value"; exit 1; }
```

Comment:
What does this change do exactly? I think nowadays we can get rid of the `watch_filter` in CAPA. It was used back in the day when we deployed CAPA components to vintage MCs, but we don't do that anymore , so there is no need to differentiate the CRs to reconcile with the watch filter.

Reply:
Ensure that errors don't get swallowed